qunetsim/backends/backend.py

```diff
@@ -57,33 +57,16 @@ def send_qubit_to(self, qubit, from_host_id, to_host_id):
         raise (EnvironmentError("This is only an interface, not \
                         an actual implementation!"))
 
-    def create_EPR(self, host_a_id, host_b_id, q_id=None, block=False):
+    def create_EPR(self, host_id, q_id=None, block=False):
```

Comment:
Does getting both parts of the EPR here improve the code?

Reply:
The idea here was that the EPR pair should not be created somewhere in the network anymore, but instead at one certain location (at a user). This is necessary for the emulation but should also improve the code.